headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -912,15 +1089,21 @@ ASSERT_SIZE(struct spawn_position, 2);
 // Dungeon floor properties
 struct floor_properties {
     struct floor_layout_8 layout; // 0x0
-    int8_t n_rooms;               // 0x1: Number of rooms to be generated
-    uint8_t tileset;              // 0x2
+    // 0x1: Affects the number of rooms to be generated. If it's positive, a slight random variation
+    // (between -2 and +1) is applied to the value (the final value must be at least 1). If it's
+    // negative, its absolute value is used without adding a random variation.
+    int8_t room_density;
```

Comment:
All the other densities also represent the number of <items/traps/enemies/whatever> to spawn (with maybe a random variation), so it would make sense to keep this consistent, right? Also, this is the name that has always been used so far when talking about floor attributes (in ppmdu, SkyTemple, documentation, etc.).

Reply:
Fair enough, we'll use the "density" standard then.